### ISSUE TITLE
How to get Svg from Resource PCL

### ISSUE THREAD

**Issue by predalpha** [luberda-molinet/FFImageLoading#515]
Hello,

im trying your library to render SVG on cross platform app.
My svg files are under my pcl project as embedded resources.
Is there a way to get them ?

Regards

**Comment by LtDrunk** [comment]
Still no answer?
I am wondering this myself..

**Comment by daniel-luberda** [comment]
`SvgImageSource.FromResource` method : https://github.com/luberda-molinet/FFImageLoading/blob/master/source/FFImageLoading.Svg.Forms/SvgImageSource.cs#L124

**Comment by michelmoorlag** [comment]
Hi Daniel,
I have used this code to load an svg from ios or Android resources folder and it works fine:

<ffimageloadingsvg:SvgCachedImage  Source="svglogo.svg" />

But it means I have to add every svg to multiple projects. Is it possible to add svg files once to the pcl project as embedded resource and load them from there by there resourceId?
I read about FormResource but how would I used that in XAML? How would I adjust the above xaml to read the svglogo.svg from a pcl?
